bigstream/features.py

```diff
@@ -40,18 +44,60 @@ def blob_detection(
         detected blobs. The last column is the image intensity at the
         detected coordinate location.
     """
-
     processed_image = np.copy(image)
-    if winsorize_limits is not None:
-        processed_image = winsorize(processed_image, limits=winsorize_limits)
+    start_time = time.time()
```

Comment:
this has an option to either use DOG or LOG. Other than that it is based on your fishspot.detect package

Reply:
Good feature addition! Personally, I believe LOG should always be superior to DOG - since mathematically speaking DOG is just an approximation to LOG anyway - but it would be faster, and some people probably want to keep the same workflow they've used in the past and not migrate to anything new, even if the new thing is technically the same as the old thing if used correctly.